src/ua/ithillet/lesson2/part3/Main.java

```diff
@@ -0,0 +1,20 @@
+package ua.ithillet.lesson2.part3;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Square sqr = new Square(5);
```

Comment:
Якщо є бажання скоротити назву, то скороти через використання `var`, а змінну краще написати повністю - краще для розуміння коду. 
```suggestion
        var square  = new Square(5);
```